core/templates/dev/head/base.html

```diff
@@ -280,6 +281,16 @@
               {{ warnings_and_loader() }}
             </div>
 
+            <noscript>
+              <div class="oppia-page-cards-container">
+                <div class="md-default-theme oppia-page-card oppia-long-text">
+                  <h2>Javascript Not Enabled</h2>
```

Comment:
Just being a little nit picky here, but displaying something like: 
`Your Javascript is not enabled :( ` 
That way the message delivers a bit more personality.